### PR TITLE
Initial commit prior to arXiv posting for EWFS

### PR DESCRIPTION
**Pull request by vprusso**
- Putting the bulk of the EWFS code, plots, and data. 
- arXiv identifier has not yet been created (so marking this as a draft for now). Once this has been generated, will update the relevant files.

### CODE REVIEW

**Comment by vprusso** [comment]
Adding @natestemen as a reviewer as he suggested the consolidation of the data files (which I think was a great suggestion).

File: ewfs/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+```
+poetry shell
+```
+
+Next, launch a local Jupyter notebook server.
+
+
+```
+jupyter notebook
+```
```
Comment (natestemen) [inline ewfs/README.md:28]: Adding a one line description of the two `ipynb` files (`demo` and `plots`) could be handy.

File: .gitignore
Comment (natestemen) [inline .gitignore:1]: My guess is someone used one of the autogenerated `.gitignore` files here, which is okay, but it adds a ton of stuff that we'll never need. Definitely not a blocker to remove, but just a bit of bloat.

**Review by natestemen — APPROVED** [review]
Approving, but I did not look at any of the python code. I only checked out the two README files and the general structure. If you want me to do a deeper review on the code as well, I can, but that will take some time.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ A repository for archived research code and tutorials on open-source quantum sof
 - [Zero noise extrapolation on logical qubits by scaling the error correction code distance](/ds_zne/). arXiv:2304.14985.
 - [Counting collisions in random circuit sampling for benchmarking quantum computers](/collisions/). arXiv:2312.04222.
 - [Quantum error mitigation by layerwise Richardson extrapolation](/lre/). arXiv:2402.04000.
+- [Towards violations of Local Friendliness with quantum computers](/ewfs/). arXiv:X.
```
[the remark this answers is not in the document]
Comment (vprusso) [reply README.md:9]: Done!

File: ewfs/README.md (hunk 1 above)
Comment (vprusso) [reply ewfs/README.md:28]: Good point. Done!

File: .gitignore
Comment (vprusso) [reply .gitignore:1]: Agreed and cleaned this up. Thanks for the suggestion!

**Pull request merged**